data/changelog.txt

```diff
@@ -61,14 +62,21 @@ Adds 'What the hell happened' stories to NPCs, keyed off backgrounds.
 Feral Humans with blob psychosis.
 Allow plank and branch bundles to be made from cordage, vines, or wire.
```

Comment:
BTW, regarding feral humans, I've been trying to figure out how best to have some of the more-sapient variety. Getting them to use guns appropriately looks to be a difficulty.
```suggestion
Allows plank and branch bundles to be made from cordage, vines, or wire.
```